docs/admin/release_notes/version_2.8.md

```diff
@@ -27,7 +27,9 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - [#61](https://github.com/nautobot/nautobot-app-floor-plan/issues/61) - Fixed colors not displaying properly when using dark mode theme.
 - [#170](https://github.com/nautobot/nautobot-app-floor-plan/issues/170) - Fixed default label tab not being active when validation error occurs.
 - [#173](https://github.com/nautobot/nautobot-app-floor-plan/issues/173) - Fixed bullets to render properly in mkdocs which expects 4 spaces to indent.
+- [#180](https://github.com/nautobot/nautobot-app-floor-plan/issues/180) - Fixed bugs where locations without Floorplans would not load Power Feed detail view, and weight attribute for buttons had string configured.
```

Comment:
This should have gone to a new sections for 2.8.1, same with the Rebaked fragment below.